src/alfasim_sdk/alfasim_sdk_api/api.h

```diff
@@ -620,6 +620,26 @@ DLL_EXPORT int get_deposition_thickness(
    void* ctx, double** out, int phase_id, enum TimestepScope ts_scope, int* size
 );
 
+/*!
+    Logs a warning message at the solver logger.
+
+    @param[in] ctx ALFAsim's plugins context.
+    @param[in] plugin_id ALFAsim's plugins id
+    @param[in] warning_message Warning Message.
+    @return An #error_code value.
+*/
+DLL_EXPORT int log_warning_message(void* ctx, const char* plugin_id, const char* warning_message);
+
+/*!
+    Logs an information message at the solver logger.
```

Comment:
What is the difference between warning and information? When and where is each one shown?
Perhaps, It would be more informative to the plugin dev explaining these differences in the docs.

Reply:
The Warning logs the message in the following structure: "Warning: " + warning_message while the Information Message logs the information message directly.
I can put that in the doc